repository_service_tuf_worker/services/keyvault/local.py

```diff
@@ -20,31 +20,63 @@ class KeyVaultError(Exception):
 class LocalKeyVault(IKeyVault):
     """Local KeyVault type"""
 
-    def __init__(self, path: str):
+    def __init__(
+        self,
+        path: str,
+        online_key_name: Optional[str] = "online.key",
+        online_key_pass: Optional[str] = None,
+        online_key_type: Optional[str] = "ed25519",
```

Comment:
I was thinking now about this configuration, and to start, we should limit the support and extend it by demand.
We should start supporting only two key cryptography types. My suggestion 'ed25519' and 'RSA'
Otherwise, in the [Task: Remove LocalKeyVault functionality to write keys in the local storage](https://github.com/vmware/repository-service-tuf-worker/issues/170) we will add a lot of algorithms.
I suggest we add it as an `Optional` `enum` list with the default as `ed25519`

Reply:
This was a topic of a long discussion.
We decided to validate the type and algorithm of the keys during the bootstrap ceremony and later when implemented during the rotation of the key as well.
We decided this to be a separate issue: https://github.com/vmware/repository-service-tuf-worker/issues/202